server/controllers/finance/receipts/list.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const path = require('path');
+const BadRequest = require('../../../lib/errors/BadRequest');
+const supportedRender = {
+  json : require('../../../lib/renderers/json'),
+  html : require('../../../lib/renderers/html'),
+  pdf  : require('../../../lib/renderers/pdf')
+};
+
+const defaultRender = 'pdf';
+const template = path.normalize('./server/controllers/finance/receipts/list.handlebars');
+const receiptOptions = { pageSize : 'A4', orientation: 'landscape' };
+
+// export the receipt object
+exports.build = build;
+
+/**
+ * @function build
+ * @desc build a report for invoice patient report of metadata
+ * @param {array} data invoice patient report of metadata
+ * @return {object} promise
+ */
+function build(data, request) {
+
+
+  let queryString  = request.query;
+  let renderTarget = (queryString && queryString.renderer) ? queryString.renderer : defaultRender;
```

Comment:
Just a comment (no need to change this line of code), but you could do the same thing like this:
```js
const renderTarget = queryString && queryString.renderer || defaultRender;
```
Pretty cool!